src/rpc/mining.cpp

```diff
@@ -126,13 +126,16 @@ static UniValue generatetoaddress(const JSONRPCRequest& request)
 
 UniValue getnewblock(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() > 2)
+    if (request.fHelp || request.params.size() > 3)
         throw std::runtime_error(
                 "getnewblock\n"
                 "\nGets hex representation of a proposed, unmined new block\n"
                 "\nArguments:\n"
-                "1. address         (string, required) The address to send fees and the newly generated coin.\n"
-                "2. required_age    (numeric, optional, default=0) How many seconds a transaction must have been in the mempool to be inluded in the block proposal. This may help with faster block convergence among functionaries using compact blocks.\n"
+                "1. address                    (string, required) The address to send fees and the newly generated coin.\n"
+                "2. required_age               (numeric, optional, default=0) How many seconds a transaction must have been in the mempool to be inluded in the block proposal. This may help with faster block convergence among functionaries using compact blocks.\n"
+                "3. xfield_type:new_xfield_value (multiple, optional) Change network wide value of the xfield parameter from the next block. The xfield parameter is identified by the xfield_type.\n"
+                " xfield_type supported are:\n"
+                "    1 : Aggregate Public Key of the Tapyrus Signer network.  (string )\n"
                 "\nResult\n"
                 "blockhex      (hex) The block hex\n"
                 "\nExamples:\n"
```

Comment:
Can you add `HelpExampleCli` about xfield option.

Reply:
done